sormas-backend/src/main/java/de/symeda/sormas/backend/report/WeeklyReportFacadeEjb.java

```diff
@@ -205,6 +206,10 @@ public List<WeeklyReportRegionSummaryDto> getSummariesPerRegion(EpiWeek epiWeek)
 		return summaryDtos;
 	}
 
+	public boolean isNationOrNoneJurisdiction(JurisdictionLevel jurisdictionLevel) {
```

Comment:
I don't see a need to have a separate method for this because it's only used in one place; just move the check to the if clause above please. :) (same for the other class)

Reply:
Done :)